content/en/docs/concepts/glossary.md

```diff
@@ -9,189 +9,369 @@ others. This page captures terminology used in the project and what it means.
+
+### **[Receiver]**
+
+Term used by the [`Collector`](/docs/collector/configuration/#receivers) to
+define how telemetry data is received. Receivers can be push- or pull-based.
+
+### **Request**
+
+See [`Distributed Tracing`](#distributed-tracing).
+
+### **Resource**
+
+Captures information about the entity for which telemetry is recorded. For
+example, a process producing telemetry that is running in a container on
+Kubernetes has a pod name, it is in a namespace and possibly is part of a
+deployment which also has a name. All three of these attributes can be included
+in the `Resource` and applied to any data source.
+
+### **[REST][]**
+
+Short for Representational State Transfer.
+
+### **[RPC][]**
+
+Short for Remote Procedure Call.
+
+### **[Sampling][]**
+
+A mechanism to control the amount of data exported. Most commonly used with the
+[`Tracing`](#trace) [`Data Source`](#data-source).
+
+### **SDK**
+
+Short for Software Development Kit. Refers to a telemetry SDK that denotes a
+[`Library`](#library) that implement the OpenTelemetry [`API`](#api).
+
+### **Semantic Conventions**
+
+Defines standard names and values of [`Metadata`](#metadata) in order to provide
+vendor-agnostic telemetry data.
+
+### **Service**
+
+A component of an [`Application`](#application). Multiple instances of a
+[`Service`](#service) are typically deployed for high availability and
+scalability. A [`Service`](#service) may be deployed in multiple locations.
+
+### **[Span][]**
+
+Represents a single operation within a [`Trace`](#trace).
+
+### **Span Link**
+
+A span link is a link between causally-related spans. For details see
+[Links between spans](/docs/reference/specification/overview#links-between-spans)
+and
+[Specifying Links](/docs/reference/specification/trace/api#specifying-links).
+
+### **[Specification][]**
+
+Describes the cross-language requirements and expectations for all
+implementations.
+
+### **[Status][]**
+
+The result of the operation. Typically used to indicate whether an error
+occurred.
+
+### **Tag**
+
+see [`Metadata`](#metadata).
+
+### **[Trace][]**
+
+A [`DAG`](#dag) of [`Spans`](#span), where the edges between [`Spans`](#span)
+are defined as parent/child. relationship.
+
+### **[Tracer][]**
+
+Responsible for creating [`Spans`](#span).
+
+### **Transaction**
+
+See [`Distributed Tracing`](#distributed-tracing).
+
+### **[zPages][]**
+
+An in-process alternative to external exporters. When included, they collect and
+aggregate tracing and metrics information in the background; this data is served
+on web pages when requested.
 
 ## Additional Terminology
 
 ### Traces
 
-- **[Trace API
-  Terminology]({{< relref "/docs/reference/specification/trace/api" >}})**
-- **[Trace SDK
-  Terminology]({{< relref "/docs/reference/specification/trace/sdk" >}})**
+##### **[Trace API Terminology](/docs/reference/specification/trace/api)**
+
+##### **[Trace SDK Terminology](/docs/reference/specification/trace/sdk)**
```

Comment:
I don't think that these sub-sub-entries under **Additional Terminology** need to be headings, in particular since they are links to other resources. I'd leave these as lists.